lib/components/viewers/nearby/nearby-filter.tsx

```diff
@@ -0,0 +1,77 @@
+import { Check2 } from '@styled-icons/bootstrap/Check2'
+import { Parking } from '@styled-icons/fa-solid/Parking'
+
+import { ComponentContext } from '../../../util/contexts'
+import { getBaseColor } from '../../util/colors'
+import { invisibleCss } from '@opentripplanner/trip-form/lib/MetroModeSelector'
```

Comment:
Don't import from /lib/ directly. If this isn't exported then it needs to be exported in an otp ui update

Reply:
Good solution! I agree